tools/clang-format-fix.sh

```diff
@@ -2,6 +2,4 @@
 
 set -e
 
-clang-format --version | cut -d ' '  -f 3 | grep -q "^5\." || (echo "Requires clang-format version 5" && exit 1)
```

Comment:
Because we don't need to depend on the version with an exhaustive enough configuration. Not all platforms ship clang-format v5. Arch is at v7 right now, non of the Debians are at v5 either (stretch is at v3.9 and buster is at v6).

Reply:
But if you are on `clang-format-5` and use `clang-format-7` you will receive errors
That is reason why it was fixed on version